src/asn1-parser.spec.ts

```diff
@@ -28,22 +28,41 @@ describe("parsePublicKey", () => {
 });
 
 describe("parseSignature", () => {
-  it("ok", () => {
+  it("ok with 142-length signature", () => {
     const buf = Buffer.from(
       "304502201fa98c7c5f1b964a6b438d9283adf30519aaea2d1a2b25ac473ac0f85d6e08c0022100e26f7c547cf497959af070ec7c43ebdbb3e4341395912d6ccc950b43e886781b",
       "hex"
     );
     const signature = parseSignature(buf);
 
+    expect(signature.r.toString("hex")).toBe(
+      "1fa98c7c5f1b964a6b438d9283adf30519aaea2d1a2b25ac473ac0f85d6e08c0"
+    );
+
+    expect(signature.s.toString("hex")).toBe(
+      "e26f7c547cf497959af070ec7c43ebdbb3e4341395912d6ccc950b43e886781b"
+    );
+
     expect(signature.r.length).toBe(32);
     expect(signature.s.length).toBe(32);
+  });
+
+  it("ok with 140-length signature", () => {
```

Comment:
:memo:
https://runkit.com/odanado/61155266fdb6be001a2223e4

Reply:
I can't understand this comment what this means 😅 Coud you explain again?

Reply:
I'm sorry.

This is just a note for myself to help me understand the principle of the original problem (#281). This is not a review comment, so don't worry about it.